Add files via upload (#2)

PavelCherkunov opened the pull request.

PavelCherkunov committed:
Add files via upload

PavelCherkunov committed:
Add files via upload

Добавление mainwindow.cpp

PavelCherkunov committed:
Add files via upload

Добавление mainwindow.cpp
Прошлое добавление - mainwindow.h, а не mainwindow.cpp

vasilij-inelsy commented on mainwindow.cpp:
```diff
@@ -0,0 +1,167 @@
+
+    Time = 0;
+    Timer = new QTimer(this);
+    connect(Timer, SIGNAL(timeout()), this, SLOT(TimerSlot()));
```
Переходи на новый синтаксис, плиз

vasilij-inelsy replied:
TimerSlot - его не видно в это ПРе. А должно.
И второе - лучше все с маленькой буквы.

PavelCherkunov committed:
Update mainwindow.cpp

Переименовал переменные
Поменял синтаксис коннекта таймера для секундомера
задал цвета через переменные с именем

PavelCherkunov committed:
Update mainwindow.h
